src/main/java/thecodex6824/thaumicaugmentation/core/ThaumicAugmentationCore.java

```diff
@@ -42,64 +42,63 @@
 @SortingIndex(1005)
 @TransformerExclusions("thecodex6824.thaumicaugmentation.core")
 public class ThaumicAugmentationCore implements IFMLLoadingPlugin {
-    
-    private static Logger log = LogManager.getLogger(ThaumicAugmentationAPI.MODID + "core");
-    
+
+    private static final Logger log = LogManager.getLogger(ThaumicAugmentationAPI.MODID + "core");
+
     private static Configuration config;
     private static boolean enabled;
     private static ImmutableSet<String> excludedTransformers;
-    
+
     public ThaumicAugmentationCore() {
-        if (config != null)
-            throw new RuntimeException("Coremod loading twice (?)");
-        
-        config = new Configuration(new File("config", ThaumicAugmentationAPI.MODID + ".cfg"));
-        enabled = !config.getBoolean("DisableCoremod", "general", false, "");
-        excludedTransformers = ImmutableSet.copyOf(config.getStringList("DisabledTransformers", "general", new String[0], ""));
     }
-    
+
     public static Logger getLogger() {
         return log;
     }
-    
+
     public static boolean isEnabled() {
         return enabled;
     }
-    
+
     public static Configuration getConfig() {
         return config;
     }
-    
+
     public static Set<String> getExcludedTransformers() {
+        if (config == null) {
+            config = new Configuration(new File("config", ThaumicAugmentationAPI.MODID + ".cfg"));
+            enabled = !config.getBoolean("DisableCoremod", "general", false, "");
```

Comment:
I don't think this can be here, since the transformer checks the enabled flag before getting the excluded transformers. That means that that flag will always be false, and nothing will get loaded. It probably has to go in injectData.